src/admindoc/setup/elasticsearch-setup.md

```diff
@@ -746,21 +746,12 @@ This can be done using a customized Luke tool and looking at the Lucene index le
 # view indexed tokens for dc:title.fulltext of document 3d50118c-7472-4e99-9cc9-321deb4fe053
 curl -XGET 'localhost:9200/nuxeo/doc/_search?search_type=count&pretty' -d'{
  "query" : {"ids" : { "values" : ["3d50118c-7472-4e99-9cc9-321deb4fe053"] }},
- "aggs": {"my_aggs": {"terms": {"field": "dc:title.fulltext", "order" : { "_count" : "desc" }}}}}'
+ "aggs": {"my_aggs": {"terms": {"field": "dc:title.fulltext", "order" : { "_count" : "desc" }, "size": 1000}}}}}}'
 ```
 
 &nbsp;
 
-The above query shows only the first 10 results. One can set the result size by adding the size option:
-
-```bash
-# view indexed tokens for dc:title.fulltext of document 3d50118c-7472-4e99-9cc9-321deb4fe053
-curl -XGET 'localhost:9200/nuxeo/doc/_search?search_type=count&pretty' -d'{
- "query" : {"ids" : { "values" : ["3d50118c-7472-4e99-9cc9-321deb4fe053"] }},
- "aggs": {"my_aggs": {"terms": {"field": "dc:title.fulltext", "order" : { "_count" : "desc" }, "size": 1000}}}}}'
-```
-
-&nbsp;
+The above query shows only the first 1000 tokens. (One can set the result size by adding the size option)
```

Comment:
remove the sentence "(One can set the result size by adding the size option)" you already added the "size" option, or even simplier update the comment above the query.